waspc/src/Wasp/Generator/WaspLibs/IO.hs

```diff
@@ -0,0 +1,59 @@
+module Wasp.Generator.WaspLibs.IO
+  ( copyWaspLibs,
+  )
+where
+
+import Control.Exception (catch)
+import StrongPath
+  ( Abs,
+    Dir,
+    Path',
+    castRel,
+    fromAbsDir,
+    (</>),
+  )
+import System.Directory (createDirectory)
+import Wasp.AppSpec (AppSpec)
+import qualified Wasp.AppSpec as AS
+import Wasp.Generator.Common (ProjectRootDir)
+import Wasp.Generator.Monad (GeneratorError (GenericGeneratorError))
+import Wasp.Generator.WaspLibs.Common
+  ( LibsRootDir,
+    LibsSourceDir,
+    getAbsLibsSourceDirPath,
+    libsRootDirInGeneratedCodeDir,
+    libsRootDirNextToSdk,
+  )
+import qualified Wasp.Generator.WaspLibs.WaspLib as WaspLib
+import Wasp.Util.IO (copyFile, deleteDirectoryIfExists)
+
+copyWaspLibs :: AppSpec -> Path' Abs (Dir ProjectRootDir) -> [WaspLib.WaspLib] -> IO (Maybe [GeneratorError])
+copyWaspLibs spec dstDirPath waspLibs = do
```

Comment:
Can we give richer name to `dstDirPath`? It is hard to follow what it is in the rest of the code.